### PR TITLE
[Dijkstra] fixing PLSQL #2135

### PR DESCRIPTION
**Pull request by RediIssac**
Fixes # .2135

Changes proposed in this pull request:
-
-
-

@pgRouting/admins

### CODE REVIEW

**Comment by cvvergara** [comment]
Please modify the titel of your PR

**Comment by cvvergara** [comment]
Should look like the title on #2149

File: docqueries/dijkstra/doc-pgr_dijkstra.result [hunk 1]
```diff
@@ -81,8 +81,8 @@ SELECT * FROM pgr_dijkstra(
   10 |        3 |        11 |       5 |    5 |   -1 |    0 |        2
 (10 rows)
 
--- q6
--- q7
+/* -- q6 */
```
Comment (cvvergara) [inline docqueries/dijkstra/doc-pgr_dijkstra.result:84]: Please manually add 2 blank lines
Should look like this.
```
/* -- q6 */


/* -- q7 */
```
Make sure that you are not introducing the windows end of line

File: docqueries/dijkstra/doc-pgr_dijkstra.result [hunk 2]
```diff
@@ -216,8 +216,8 @@ SELECT * FROM pgr_dijkstra(
   18 |        5 |        11 |       5 |    5 |   -1 |    0 |        4
 (18 rows)
 
--- q8
--- q9
+/* -- q8 */
```
Comment (cvvergara) [inline docqueries/dijkstra/doc-pgr_dijkstra.result:219]: Same here

File: docqueries/dijkstra/doc-pgr_dijkstra.result [hunk 3]
```diff
@@ -329,8 +329,8 @@ SELECT * FROM pgr_dijkstra(
   10 |        3 |        11 |       5 |    5 |   -1 |    0 |        2
 (10 rows)
 
--- q10
--- q11
+/* -- q10 */
+/* -- q11 */
```
Comment (cvvergara) [inline docqueries/dijkstra/doc-pgr_dijkstra.result:333]: Same here

File: docqueries/dijkstra/doc-pgr_dijkstra.result [hunk 4]
```diff
@@ -405,8 +405,8 @@ SELECT * FROM pgr_dijkstra(
    2 |        2 |         2 |       5 |    5 |   -1 |    0 |        1
 (2 rows)
 
--- q12
--- q13
+/* -- q12 */
+/* -- q13 */
```
Comment (cvvergara) [inline docqueries/dijkstra/doc-pgr_dijkstra.result:409]: same here

File: docqueries/dijkstra/doc-pgr_dijkstra.result [hunk 5]
```diff
@@ -526,8 +526,8 @@ SELECT * FROM pgr_dijkstra(
   12 |        3 |        11 |       5 |    5 |   -1 |    0 |        2
 (12 rows)
 
--- q14
--- q15
+/* -- q14 */
+/* -- q15 */
```
Comment (cvvergara) [inline docqueries/dijkstra/doc-pgr_dijkstra.result:530]: same here

File: docqueries/dijkstra/doc-pgr_dijkstra.result [hunk 6]
```diff
@@ -629,8 +629,8 @@ SELECT * FROM pgr_dijkstra(
    6 |        6 |         2 |       3 |    3 |   -1 |    0 |        5
 (6 rows)
 
--- q16
--- q17
+/* -- q16 */
```
Comment (cvvergara) [inline docqueries/dijkstra/doc-pgr_dijkstra.result:632]: same here

File: docqueries/dijkstra/doc-pgr_dijkstra.result [hunk 7]
```diff
@@ -686,8 +686,8 @@ SELECT * FROM pgr_dijkstra(
    2 |        2 |         2 |       3 |    3 |   -1 |    0 |        1
 (2 rows)
 
--- q18
--- q19
+/* -- q18 */
+/* -- q19 */
```
Comment (cvvergara) [inline docqueries/dijkstra/doc-pgr_dijkstra.result:690]: same here

**Review by cvvergara — CHANGES_REQUESTED** [review]

File: docqueries/dijkstra/doc-pgr_dijkstra.result (hunk 7 above)
Comment (RediIssac) [reply docqueries/dijkstra/doc-pgr_dijkstra.result:704]: I have added two blank lines manually and pushed  new iteration.

**Review by cvvergara — APPROVED** [review]
Thanks for the contribution

**Pull request merged**